microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -995,12 +1072,42 @@ Status:
 _new as of Microsoft Edge 118_
 
 Whether to enable the Storage Buckets Tree in the **Application** tool.
-<!-- todo: need text -->
+<!-- todo old: need text -->
 
 <!-- 
 See also:
 * []() -->
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 120.
-* This checkbox is present in Microsoft Edge Stable 118.
+* This checkbox is present in Microsoft Edge Canary 121.
+* This checkbox is present in Microsoft Edge Stable 119.
+
+
+<!-- ====================================================================== -->
+## Enable context menu that allows to modify trees in the Flame Chart
+<!-- no (?) link -->
+
+_new as of Microsoft Edge 121_
+
+Adds a right-click menu that allows modifying trees in the flame chart in the **Performance** tool.  Menuitems:
+* **Merge function**
+* **Collapse function**
+* **Collapse repeating descendants**
+* **Undo collapse function**
+* **Undo collapse repeating descendants**
+* **Undo all actions**
+* **Load profile**
+* **Save profile**
+
+With this experiment enabled, you can right-click events in the flame chart of the **Performance** tool, and modify the chart at that location.
+
+![Context menu to modify flame chart trees](./index-images/context-menu-modify-flame-chart-trees.png)
```

Comment:
@captainbrosset - can you create a final png, if Edge now has this feature?
rendered:
https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/experimental-features/?branch=pr-en-us-2951#enable-context-menu-that-allows-to-modify-trees-in-the-flame-chart

Reply:
Will push a commit with a new image.